test/java/seedu/addressbook/utils/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.utils;
+
+import seedu.addressbook.common.Utils;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.util.Vector;
+
+public class UtilsTest {
+    
+    @Test
+    public void test_isAnyNull_allNotNull() {
```

Comment:
`test_` is not necessary. After removing it, the name should have three parts `methodBeingTested_scenario_expectedResult`